schema/fs.go

```diff
@@ -202,227 +202,310 @@ func _escFSMustString(useLocal bool, name string) string {
 
 var _escData = map[string]*_escFile{
 
-	"/.content-descriptor.json.un~": {
-		local:   ".content-descriptor.json.un~",
-		size:    1548,
-		modtime: 1485467092,
+	"/.fs.go.un~": {
```

Comment:
More local files.  I think you should land #538 (I've added a comment reproducing my successfully ignoring local files like this to that PR) or remove your local editor files before rebuilding `fs.go`.